src/context/box-repository.ts

```diff
@@ -29,6 +30,10 @@ export class BoxRepository extends BaseRepository<BoxEntity> {
 
   public override find(options: BoxFindOptions): Promise<BoxEntity[]> {
     const { spent, tokenId, registers } = options;
+    if (options.where?.address) {
+      options.where.ergoTree = Address.fromBase58(options.where.address).ergoTree;
```

Comment:
What happens if both `address` and `ergoTree` are specified? 
This way, `address` overwrites `eroTree`. Is that the behavior we want? Shouldn't we explicitly forbid this kind of parameters?